example/index.js

```diff
@@ -107,5 +107,14 @@ manager.on('maxRolesReach', async (member, userAction, nbRoles, maxRoles) => {
 manager.on('interaction', (rte, interaction) =>
   console.log(`An interaction has been made by ${interaction.member.displayName}`)
 );
+manager.on('conditionsNotMet', async (member,
+  userAction,
+  role,
+  dependencies) => {
+  console.log(`${member.displayName} doesn't have the required dependencies to get the role ${role}!`, dependencies);
+  userAction instanceof ButtonInteraction && await userAction.editReply({
+    content: `${member.displayName} doesn't have the required dependencies to get the role ${role}!`
+  });
+});
```

Comment:
With this, you listen to the `conditionsNotMet` event but there are currently no channel registered that would trigger this behavior. Could you add update the latest item of the array of `rolesToEmojis` (declared at line 20 of this file) so we can have a role that have dependency on other(s) please?

Should look like something belong the lines of 
```js
      ,
      {
        emoji: '<:EMOJI_NAME:EMOJI_ID>',
        role: 'ROLE_ID',
        requiredRoles: [ 'ROLE_ID' ]
      }
```